### PR TITLE
keep the original command while in remote mode

### LINKED ISSUE #572: Run the command when in remote mode
**Is your feature request related to a problem? Please describe.**
I want okteto to still run the command when running in --remote mode

**Describe alternatives you've considered**
Run the command manually after

### PR DESCRIPTION
**Pull request by rberrelleza**
Fixes #572 

## Proposed changes
- update start.sh so to use the https://github.com/okteto/supervisor instead of launching processes
- keep the original command for remote

### CODE REVIEW

**Comment by rberrelleza** [comment]
@pchico83 do you think we should add a healthcheck, so the container dies if the SSH server dies? Other option could be to use a process manager as the command, and have that launch syncthing and remote (https://github.com/ochinchina/supervisord looks interesting).

**Comment by pchico83** [comment]
@rberrelleza I would go with https://github.com/ochinchina/supervisord, but I am fine with the other approaches for now. The supervisor ensures the pod is not restarted, which cleans all data in memory or not belonging to the docker image. And who knows, we might need more processes in the future.

**Comment by rberrelleza** [comment]
Updated with the supervisor change. @pchico83 can you give it a pass?

File: pkg/model/dev_test.go [hunk 1]
```diff
@@ -298,8 +298,8 @@ func Test_LoadRemote(t *testing.T) {
 
 	dev.LoadRemote()
 
-	if dev.Command[0] != "/var/okteto/bin/remote" {
-		t.Errorf("remote command wasn't set: %s", dev.Command)
+	if dev.Command[0] != "uwsgi" {
```
Comment (pchico83) [inline pkg/model/dev_test.go]: what is this for?

Comment (rberrelleza) [reply pkg/model/dev_test.go]: test to validate that the command is not replaced when remote mode is enabled.

**Pull request merged**